src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -246,6 +246,7 @@
         "TOGGLED_SUCCESS": "Toggled project successfully.",
         "FAILED_TO_DELETE_PROJECT": "Project contains repositories or replication rules or helm-charts cannot be deleted.",
         "INLINE_HELP_PUBLIC": "When a project is set to public, anyone has read permission to the repositories under this project, and the user does not need to run \"docker login\" before pulling images under this project.",
+        "PROXY_CACHE_BANDWIDTH":"Set the maximum network bandwidth to pull image from upstream for proxy-cache. For unlimited bandwidth, please enter -1. ",
```

Comment:
this string needs to be added to all language files. (Put it in the english version), for later translation.

 Its than easier to pick it up by others and translate

Reply:
Added this line to all the language files